strictdoc/export/html/_static/stable_uri_forwarder.js

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Here, please add one-two sentences about the purpose if this JS file. Nothing fancy, just a small description.